pkg/integrations/install/install.go

```diff
@@ -30,10 +31,11 @@ import (
 	// v2 integrations
 	//
 
-	_ "github.com/grafana/agent/pkg/integrations/v2/agent" // register agent
-	_ "github.com/grafana/agent/pkg/integrations/v2/apache_http"
+	_ "github.com/grafana/agent/pkg/integrations/v2/agent"              // register agent
+	_ "github.com/grafana/agent/pkg/integrations/v2/apache_http"        // register apache_exporter
 	_ "github.com/grafana/agent/pkg/integrations/v2/app_agent_receiver" // register app_agent_receiver
-	_ "github.com/grafana/agent/pkg/integrations/v2/eventhandler"
-	_ "github.com/grafana/agent/pkg/integrations/v2/snmp_exporter"
-	_ "github.com/grafana/agent/pkg/integrations/v2/vmware_exporter"
+	_ "github.com/grafana/agent/pkg/integrations/v2/blackbox_exporter"  // register blackbox_exporter
```

Comment:
Nice cleanup job for the various integration! 

Super nitpicky comment here, but we're registering the integration with the `blackbox` name (same for some of the others), doesn't make it any difference tbh.